seeds/_sector/dex/pharaoh_avalanche_c_base_trades_seed.csv

```diff
@@ -0,0 +1,2 @@
+blockchain,project,version,block_date,tx_hash,evt_index,token_bought_address,token_sold_address,block_number,token_bought_amount_raw,token_sold_amount_raw
+avalanche_c,pharaoh,1,2023-12-27,0xba5d58703f53313a5d46bc2f14043735580123c66817558adfee943fe5fb5e49,19,0x9702230A8Ea53601f5cD2dc00fDBc13d4dF4A8c7,0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E,39584150,50008753,50000000
```

Comment:
Hi @discochuck, I think this line is a bit tricky.
As the swap event showed `50008753`:
https://snowtrace.io/tx/0xba5d58703f53313a5d46bc2f14043735580123c66817558adfee943fe5fb5e49?chainId=43114#eventlog
However the result in model is `50008754`:
https://dune.com/queries/3330264

Do you have any idea why?

Reply:
right out of the gate, no. 

tho, I'm wondering if it could have to do with Odos (aggregator) handling our swaps? but still not sure that would account for such a small amount.. 

so, to sum it it up.. no. I have no idea.

Reply:
crap, the image didn't copy . lets see:
![pharaoh-swap-kimbo-odos](https://github.com/duneanalytics/spellbook/assets/1681165/c7aea00a-f97e-4b9c-8299-5ea7ecf0a60c)

you can see 0.000000064 kimbo go to Odosrouterv2 (.25 avax for 57,015.0772854041 Kimbo)

Reply:
![pharaoh-swap-50-usdc-50-usdt-odos](https://github.com/duneanalytics/spellbook/assets/1681165/763d9f84-f6a9-44df-b0f5-0f6c9fa5d887)

Here's the net transfers from the provided Seed transaction as a better example.. It looks like that missing 1/10000th of a cent went to Odosrouterv2.

Reply:
Do you think we may keep the current model implementation and modify the seed?
Or we can keep the seed and modify the model implementation(maybe one or two lines)

Reply:
I think the seed is right, and the implementation should be modified, tho admittedly that's a bit beyond my ability, so any assistance would be greatly appreciated.

Reply:
I will see how to modify the implementation.

Reply:
We can see the above trades are recorded in glacier.trades:
https://dune.com/queries/3332931

With the correct number matching the seed record.

Hence there are 2 questions:
1. Is pharaoh an aggregator? (In this case we may need to put it into dex_aggregator.trades)
2. There are trades covered by `glacier.trades` in this model, do we need to exclude them from `pharaoh.trades` to avoid duped records?

Reply:
glacier was the initial version before we upgraded to ramses/uni v3 tech stack.

so it would track that the trades could be found there.

Reply:
We may need to reimplement the model, using union to connect two parts above.